src/libcompiler_builtins/Cargo.toml

```diff
@@ -13,3 +13,4 @@ core = { path = "../libcore" }
 
 [build-dependencies]
 gcc = "0.3.27"
+rustc-cfg = { git = "https://github.com/japaric/rustc-cfg", branch = "llvm-target" }
```

Comment:
I was talking to @japaric about this the other day. This basically does what the extra env vars from https://github.com/rust-lang/rfcs/pull/1721 would do, so it can be considered a stop-gap perhaps.